bundles/GraphicServiceBilling/Controller/MainController.php

```diff
@@ -32,15 +32,13 @@ class MainController extends AbstractController
     /**
      * @Route("", name="index")
      *
-     * @param \Symfony\Component\HttpFoundation\Request                   $request
-     * @param \App\Service\MenuService                                    $menuService
-     * @param \GraphicServiceBilling\Service\GraphicServiceBillingService $graphicServiceBillingService
      * @param $boundProjectId
      *
      * @return \Symfony\Component\HttpFoundation\Response
      *
      * @throws \PhpOffice\PhpSpreadsheet\Exception
      * @throws \PhpOffice\PhpSpreadsheet\Writer\Exception
+     * @throws \Exception
```

Comment:
Can we make this a more specific exception?

Reply:
Cleaned up